### DISCUSSION TITLE
MixedContentDataCollectionError and Content Layer

### DISCUSSION THREAD

**Discussion by dustinlacewell** [withastro/roadmap#1007] [Proposal]
# Summary

Restrictions around how the `content/` folder are too strict when only utilizing new Content Layer collections.

# Background & Motivation

With new content layer loaders like glob, it makes more sense to colocate related data and content. However, if you try doing this inside the `content/` folder, Astro refuses to build.

# Goals

Maybe MixedContentDataCollectionError can do a better job at discerning whether any legacy content collections are actually involved when it detects collections sharing a root?

**Comment by florian-lefebvre** [comment]
I believe this is temporary. This won't be a limitation once Content Layer is stabilized
